File: android/app/src/main/java/com/lanbridge/app/core/models/DeviceInfo.kt
package com.lanbridge.app.core.models

import com.lanbridge.app.core.enums.TransportType

/**
 * 发现的设备信息
 */
data class DeviceInfo(
    /** 设备名称 */
    val name: String,
    /** IP 地址 */
    val ip: String,
    /** 服务端口 */
    val port: Int,
    /** 传输类型 */
    val transport: TransportType = TransportType.Udp
)

File: windows/LanAudioBridge.Desktop/Core/Models/DeviceInfo.cs
namespace LanAudioBridge.Core;

/// <summary>
/// 发现的设备信息
/// </summary>
public struct DeviceInfo
{
    /// <summary>设备名称</summary>
    public string Name;

    /// <summary>IP 地址</summary>
    public string Ip;

    /// <summary>服务端口</summary>
    public int Port;

    /// <summary>传输类型</summary>
    public TransportType Transport;
}
